### ISSUE TITLE
Keep getting ATOM error that should be marked as a linting error

### ISSUE THREAD

**Issue by Tokimon** [AtomLinter/linter-phpcs#227]
```
Error: Invalid point encountered! See console for details.
    at Error (native)
    at file:///C:/Users/Toke Voltelen/.atom/packages/linter-phpcs/lib/main.js:343:19
    at Array.map (native)
    at Object.<anonymous> (file:///C:/Users/Toke Voltelen/.atom/packages/linter-phpcs/lib/main.js:317:25)
    at next (<anonymous>)
    at step (C:\Users\Toke Voltelen\.atom\packages\linter-phpcs\lib\main.js:9:273)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

This error keeps popping up in certain files in the project I am working on.

And when I look in the console, it tells me that I have an error in the attached message, which in turn just points out that there is a linting error (a string that is too long):

![Atom console Error](http://content.screencast.com/users/Tokimon/folders/Jing/media/e388fb0f-94b3-446a-a7e3-025701405e00/2017-01-10_1047.png)

Any idea why this is? To me it seems that a regular linting error is being caught and displayed as an actual error.

**Comment by weirdan** [comment]
It looks like phpcs-linter is catching the error (message?) of another plugin (possibly this one  https://atom.io/packages/php-cs-fixer)

**Comment by Tokimon** [comment]
Update on this.
I keep getting the same error but on a string that is not too long but contains language specific characters:
`str_split("ÀÁÂÃÄÅàáâãäåÒÓÔÕÖØòóôõöøÈÉÊËèéêëÇçÌÍÎÏìíîïÙÚÛÜùúûüÿÑñ")`
(the line in question is 96 chars in total)

**Comment by weirdan** [comment]
It seems linter-phpcs omits `--encoding` parameter, so `phpcs` runs in ascii mode:
```
$ cat q.php
<?php
$q = str_split("ÀÁÂÃÄÅàáâãäåÒÓÔÕÖØòóôõöøÈÉÊËèéêëÇçÌÍÎÏìíîïÙÚÛÜùúûüÿÑñ");

$ phpcs -v --standard=PSR2 q.php
Registering sniffs in the PSR2 standard... DONE (42 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory .
Processing q.php [PHP => 11 tokens in 2 lines]... DONE in 1ms (0 errors, 1 warnings)

FILE: q.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 2 | WARNING | Line exceeds 120 characters; contains 125 characters
----------------------------------------------------------------------

Time: 50ms; Memory: 6Mb

$ phpcs --encoding=utf8 -v --standard=PSR2 q.php
Registering sniffs in the PSR2 standard... DONE (42 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory .
Processing q.php [PHP => 11 tokens in 2 lines]... DONE in 1ms (0 errors, 0 warnings)
```

**Comment by weirdan** [comment]
`--encoding` is available since 1.3.0RC1 (it's unlikely anyone runs version that old, but anyway).

**Comment by Tokimon** [comment]
I guess this is a duplicate of #211...

**Comment by Arcanemagus** [comment]
@Tokimon yes it is, closing as such!
